### PR TITLE
Don't render directive token from imports in design time

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #1138 

@NTaylorMullen @rynowak

### CODE REVIEW

File: src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/DesignTimeCSharpRenderer.cs [hunk 1]
```diff
@@ -110,8 +110,14 @@ public override void VisitDirectiveToken(DirectiveTokenIRNode node)
             const string TypeHelper = "__typeHelper";
 
             var tokenKind = node.Descriptor.Kind;
-            if (node.Source == null)
+            if (!node.Source.HasValue ||
+                (Context.SourceDocument.FileName != null &&
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/DesignTimeCSharpRenderer.cs]: Why is the null check needed here? Is this a workaround for some test issue?

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/DesignTimeCSharpRenderer.cs]: It is just to handle the unlikely case of both of them being null.

Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/DesignTimeCSharpRenderer.cs]: Just realized that doesn't matter. Will remove.

**Comment by rynowak** [comment]
:shipit: after removing silly null check

**Pull request merged**